package.json

```diff
@@ -22,8 +22,8 @@
     "devtools": "react-devtools"
   },
   "dependencies": {
-    "react-native-material-dropdown": "^0.7.1",
-    "react-native-material-textfield": "^0.12.0"
+    "react-native-material-dropdown": "git://github.com/welstarco/react-native-material-dropdown.git",
```

Comment:
This might cause some quirks during build processes I think, better to publish it to npm